porter/v1/system_service.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package porter.v1;
+
+import "porter/v1/prometheus_alerts.proto";
+
+// SystemService identifies a system service by its name, namespace and the type of k8s object it runs
+message SystemService {
+  // name is the name of the service
+  string name = 1;
+  // namespace is the namespace where it is deployed
+  string namespace = 2;
+  // involved_object_type is the k8s object type the service runs
+  InvolvedObjectType involved_object_type = 3;
+}
+
+// ServiceStatus represents the status of a service
+enum ServiceStatus {
+  // SERVICE_STATUS_UNSPECIFIED is the default value
+  SERVICE_STATUS_UNSPECIFIED = 0;
+  // SERVICE_STATUS_HEALTHY is the value for a fully healthy service
+  SERVICE_STATUS_HEALTHY = 1;
+  // SERVICE_STATUS_PARTIAL_FAILURE is the health status for a partially failed service
+  SERVICE_STATUS_PARTIAL_FAILURE = 2;
```

Comment:
What does partial failure mean? (Whatever it means should be added to this comment)

Reply:
And why separate between partial failure and full failure?

Reply:
We took inspiration from some of the status pages out there. AWS/GCP/Cloudflare etc.